js/packages/polyfill/player.ts

```diff
@@ -0,0 +1 @@
+export { Player } from '@react-native-community/audio-toolkit'
```

Comment:
Maybe rename this file to `react-native-community-audio-toolkit-player.tsx` (and its `.web.tsx` counterpart) or merge it with `record` in a single `react-native-community-audio-toolkit` file?

Reply:
Yeah i'm okay with that 👍